### ISSUE TITLE
[BUG]: Ratchet and Clank Size matter (PAL) timing problem in FMVs

### ISSUE THREAD

**Issue by gilderoylockhart01** [PCSX2/pcsx2#4746]
### Describe the Bug

The game "Ratchet and Clank Size matter" have an issue with it's FMVs on the PAL version where stock cyclerate will sometimes makes the FMVs to slows down and thus, causing audio glitches.

Note: +1 cyclerate solve the problem.

### Reproduction Steps

Load the savestate with the blockdump on the latest PCSX2 dev build at stock cyclerate, see the FMV slowing down at around 75% completation causing the audio to periodically stops.

### Expected Behavior

The FMV should plays flawlessly at stock cyclerate.

### PCSX2 Revision

dev-1705-g18311d6a4-windows-x86

### Operating System

Windows 10 (64bit)

### CPU

I9 9900k

### GPU

RTX 3070

### Emulation Settings

Stock EE cyclerate.

### GS Window Screenshots

The bug in action:

Stock cyclerate:

https://user-images.githubusercontent.com/90222976/132580641-0f08575b-dd2c-4ced-a4b8-10d792e6c6fe.mp4

+1 cyclerate:

https://user-images.githubusercontent.com/90222976/132580860-04d58088-d1bf-4cdc-8501-3af7ce353856.mp4

### Logs & Dumps

Blockdump of the game + savestate:

https://drive.google.com/file/d/1xyu-wpaNnodlBezZQXCY-viNiRiUDzfl/view?usp=sharing

**Comment by Squall-Leonhart** [comment]
This might be tied to your framerate being far more inconsistent at the default cyclerate reaching as low as 49.92 while with +1 the lowest is 49.96, the SPU engages time stretching as a result.

See if the problem occurs with high performance power profile, the variation looks like a lesser degree of #4419

**Comment by refractionpcsx2** [comment]
> This might be tied to your framerate being far more inconsistent at the default cyclerate reaching as low as 49.92 while with +1 the lowest is 49.96, the SPU engages time stretching as a result.
> 
> See if the problem occurs with high performance power profile, the variation looks like a lesser degree of #4419

I doubt that has anything to do with it, that's VPS (framerate you're achieving, game still runs at its original framerate internally), the audio will stretch/slow down to compensate for any framerate changes on the windows end to keep up.

**Comment by gilderoylockhart01** [comment]
Yeah refraction is right, that's actually an issue with the internal game frame-rate.

**Comment by Squall-Leonhart** [comment]
> If this wasn't working properly, it would be a problem on a LOT of games.

True, everyone would notice.

I actually think i've been seeing this bad behavior with other PAL titles, i put it down to my system being a little busy at the time but now i've been checking on my other system I've seen it in KH2 as well

**Comment by Mrlinkwii** [comment]
can this be checked on latest nightly :)

**Comment by TheTechnician27** [comment]
Bumping after almost two years to see if someone is able to test this on the latest nightly, keeping in mind that this is the PAL version of the game. This feels like it's an internal frame rate problem at a glance.
